.github/actions/issue-validator/package.json

```diff
@@ -1,7 +1,10 @@
 {
   "private": true,
-  "main": "index.js",
+  "main": "dist/index.js",
   "type": "module",
+  "scripts": {
+    "build": "pnpx @vercel/ncc -m build index.js --license licenses.txt"
```

Comment:
Should this be `npx` since you have a package-lock.json?

Reply:
Just building locally, manually in the monorepo, so this should be good. Not expecting many changes. I can change it anyway. 👍